Sources/Client.swift

```diff
@@ -45,14 +45,15 @@ open class Client {
         }
 
         self.configuration = configuration
-        self.spaceIdentifier = spaceIdentifier
+        self.spaceId = spaceId
     }
 
-    fileprivate func fetch<T: Decodable>(url: URL?, completion: @escaping (Result<T>) -> Void) -> URLSessionDataTask? {
+    // TODO: rename
```

Comment:
TODO's shouldn't be there. I'm actually struggling within the guidelines to name this method as it doesn't return data, but returns a task and passes in the asynchronous callback where data will actually be extracted. any suggestions?